### PR TITLE
README: Added usage with kubectl

### PR DESCRIPTION
**Pull request by albertvaka**
It's kind of basic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ chmod +x ./kind
 mv ./kind /some-dir-in-your-PATH/kind
 ```
 
+## Usage
+
 To use kind, you will need to [install docker].
```
Comment (neolit123) [inline README.md]: this sentence seems more relevant to the installation section.

Comment (albertvaka) [reply README.md]: Fixed!

File: README.md [hunk 2]
```diff
@@ -59,6 +62,12 @@ To create a cluster from Kubernetes source:
 Multi-node clusters and other advanced features may be configured with a config
 file, for more usage see [the docs][user guide] or run `kind [command] --help`
 
+To make `kubectl` use your local kind cluster, use:
```
Comment (BenTheElder) [inline README.md]: We already print a per-platform relevant usage when you run kind create cluster. I'd rather not duplicate that here. For example this version is now lacking windows.

Comment (albertvaka) [reply README.md]: I probably missed that when I first created a cluster... But you have a point, closing this then!

**Comment by BenTheElder** [comment]
We should prefer keeping minimal details in the readme just to illustrate: what it is / why it exists, and how simple it is to install and run, otherwise the docs site should cover just about everything else.

Duplicating content makes it less likely to stay up to date.

File: README.md (hunk 2 above)
Comment (BenTheElder) [reply README.md]: Thanks anyhow though, I appreciate documentation efforts especially :-)

**Pull request closed without merging**